src/playground/workspace.js

```diff
@@ -418,14 +418,15 @@ Entry.Workspace = class Workspace {
                 }
             }
 
+            const isBlockCodeView = Entry.playground.mainWorkspace.getMode() === 0 && (Entry.playground.getViewMode() === 'code' || Entry.playground.getViewMode() === 'variable');
```

Comment:
정말 별거 아니긴 한데
0을 바로 쓰기보다 `Entry.Workspace.MODE_BOARD`를 쓰는게 어떨까 싶습니다.
```suggestion
            const isBlockCodeView = Entry.playground.mainWorkspace.getMode() === Entry.Workspace.MODE_BOARD && (Entry.playground.getViewMode() === 'code' || Entry.playground.getViewMode() === 'variable');
```

Reply:
아주 중요한 포인트 찝어 주셨습니다. 감사해요~